stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/model/ModelType.kt

```diff
@@ -49,7 +49,10 @@ public object ModelType {
     public const val attach_mime_xlsx: String = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
     public const val attach_mime_zip: String = "application/zip"
     public const val attach_mime_mov: String = "video/mov"
-    public const val attach_mime_mp4: String = "video/mp4"
+    public const val attach_mime_quicktime: String = "quicktime"
```

Comment:
mime_type from Flutter for .MOV

Reply:
Is this full mime type? That's odd. Should be `video/quicktime`.